proto/migration/v1alpha1_to_v2.go

```diff
@@ -756,11 +756,11 @@ func BeaconState4844ToProto(st state.BeaconState) (*ethpbv2.BeaconState4844, err
 	if err != nil {
 		return nil, errors.Wrap(err, "could not get latest execution payload header")
 	}
-	var excessBlobs uint64
-	if val, err := sourceLatestExecutionPayloadHeader.ExcessBlobs(); err == nil {
-		excessBlobs = val
+	var excessDataGas []byte
+	if val, err := sourceLatestExecutionPayloadHeader.ExcessDataGas(); err == nil {
+		excessDataGas = val
 	}
-	// else ErrUnsupportedGetter, so we ignore its value
+	// else ErrUnsupportedGetter because it's a pre-4844 Payload, so we ignore its value
```

Comment:
hmm, on second thought. This should be an error. A post-4844 beacon state has no business storing payload headers that aren't 4844 compatible (i.e. with a nil `ExcessDataGas`. Will fix.